src/booleans.js

```diff
@@ -1,62 +1,107 @@
 function negate(a) {
-  // your code here
-};
+  if (a === true) {
+    return false;
+  }
+  return true;
+}
 
 function both(a, b) {
-  // your code here
-};
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
 
 function either(a, b) {
-  // your code here
-};
+  if (a || b === true) {
+    return true;
+  }
+  return false;
+}
 
 function none(a, b) {
-  // your code here
-};
+  if (!a && !b){
+    return true
+  }
+  return false 
+}
 
 function one(a, b) {
-  // your code here
-};
+  if ((a && !b) || (!a && b)){
+    return true;
+  }
+  return false
+}
 
 function truthiness(a) {
-  // your code here
-};
+  return Boolean(a)
+}
 
 function isEqual(a, b) {
-  // your code here
-};
+  if (a === b){
+    return true
+  }
+  return false;
+}
 
 function isGreaterThan(a, b) {
-  // your code here
-};
+  if (a > b) {
+    return true;
+  }
+  return false;
+}
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
-};
+  if (a <= b) {
+    return true;
+  }
+  return false;
+}
 
 function isOdd(a) {
-  // your code here
-};
+  if (a % 2) {
+    return true 
+  }
+  return false; 
+}
 
 function isEven(a) {
-  // your code here
-};
+  if (a % 2 === 0) {
+    return true
+  }
+  return false 
+}
 
 function isSquare(a) {
-  // your code here
-};
+  return Math.sqrt(a) % 1 === 0;
+}
 
 function startsWith(char, string) {
-  // your code here
-};
+  return string.startsWith(char)
+}
 
 function containsVowels(string) {
-  // your code here
-};
+  string = string.toLowerCase()
```

Comment:
Here you are trying to reassign the passed in string parameter. You will have to chose a new name to do this and declare a new variable with const.
like:
`````
const lowerCaseString = string.toLowerCase()
``````
and then use this variable in your if block.

Maybe a better and shorter way to do this here would be to use a regex pattern. 
Regular expressions are particularly useful for defining filters. Regular expressions contain a series of characters that define a pattern of text to be matched—to make a filter more specialised, or general.
You already made a "pattern" here which is the vowels and if you tweak the form a little bit it would be a regex pattern like this : /[aeiou]/i
To create or identify patterns this is a very useful website: https://regexr.com/

In the created patter /[aeiou]/i
The square brackets [] means match the characters in the brackets
The i means it is case sensitive.

Using this regex pattern you could use the match string method. 
`````
string.match(/[aeiou]/i)
``````
This will return the match when it finds one, for example in your first test we pass in the string 'cat' it will return an array of the matches which is only a so will return [a] if there is no match it will return null.
But we will need true or false values to be returned not the null or an array of the matches.
To do this we can use the double NOT operator. (!!)


`````
return !!string.match(/[aeiou]/i)
``````